### ISSUE TITLE
same FindElementByAccessibilityId

### ISSUE THREAD

**Issue by skotian2** [microsoft/WinAppDriver#1762]
Hi All,
I have script with same  FindElementByAccessibilityId ("txtBox") is there a better way to handle in WinAppDriver. I am unable to  click on the third element in the form. Any help would be highly appreciated.

**Comment by liljohnak** [comment]
We use the linq library.
`var thirdBox = window.FindElementsByAccessibilityId ("txtBox").ElementAt(2);`

**Comment by skotian2** [comment]
Thanks Jonathan - Do you have any example of passing those value (ThirdBox)  with  FindElementsByAccessibilityId

**Comment by skotian2** [comment]
Issue closed
